docs/manifest.json

```
@@ -25,9 +25,15 @@
       ["Next.js starter repo", "https://github.com/clerkinc/clerk-next-app"],
       ["React starter repo", "https://github.com/clerkinc/clerk-react-starter"],
       ["Remix starter repo", "https://github.com/clerkinc/remix-auth-starter"],
```

well OK don'T make them feel left out!!!

```suggestion
      [
      "Next.js starter repo", 
      "https://github.com/clerkinc/clerk-next-app"
      ],
      [
      "React starter repo", 
      "https://github.com/clerkinc/clerk-react-starter"
      ],
      [
      "Remix starter repo", 
      "https://github.com/clerkinc/remix-auth-starter"
      ],
```